src/segment/segmentBase.js

```diff
@@ -67,7 +67,7 @@ export const segmentsFromBase = (attributes) => {
  * @param {Object} sidx the parsed sidx box
  * @return {Object} the playlist object with the updated sidx information
  */
-export const addSegmentsToPlaylist = (playlist, sidx, baseUrl) => {
+export const addSidxSegmentsToPlaylist = (playlist, sidx, baseUrl) => {
```

Comment:
Renamed this function, as it was named terribly. It only adds sidx segments to a playlist, and we have other functions that add segments from our dash manifest to a  playlist object.